keg_elements/forms/state_field.py

```diff
@@ -0,0 +1,79 @@
+from . import SelectField
+
+states = {
```

Comment:
Would be better to use something like https://github.com/unitedstates/python-us for the list of states rather than listing them all out here?